fix(www): make GitHub icon above plugin readme clearer

kbrgl opened the pull request:
It was hard to find and isn't very clear.

<!--
  Have any questions? Check out the contributing docs at https://gatsby.app/contribute, or
  ask in this Pull Request and a Gatsby maintainer will be happy to help :)
-->

## Description
Adds 'View plugin on GitHub' next to the GitHub icon, and introduces some spacing to make it easier to find. It wasn't very clear earlier where it goes (view source on GitHub could mean either page source or plugin source.)

Just a minor something I've clashed with a few times 😄 
<!-- Write a brief description of the changes introduced by this PR -->
<img width="781" alt="screen shot 2019-02-27 at 2 59 46 am" src="https://user-images.githubusercontent.com/13417496/53447728-d571d780-3a3b-11e9-9ad7-81d3d0669c59.png">

m-allanson commented on www/src/components/package-readme.js:
```diff
@@ -40,9 +42,8 @@ const PackageReadMe = props => {
-          title={`View source on GitHub`}
         >
-          <GithubIcon focusable="false" style={{ verticalAlign: `text-top` }} />
+          <GithubIcon focusable="false" style={{ verticalAlign: `text-top` }} />&nbsp;&nbsp;<span>View plugin on GitHub</span>
```
Can you remove the `&nbsp;&nbsp;` and instead add the spacing with CSS?

kbrgl replied:
Yep, no problem!

m-allanson commented on www/src/components/package-readme.js:
```diff
@@ -40,9 +42,8 @@ const PackageReadMe = props => {
           }}
           href={githubUrl}
           aria-label={`${packageName} source`}
```
Maybe remove this `aria-label` and replace it with an [`aria-labelledby` attribute](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-labelledby_attribute) that links to the text `<span>View plugin on GitHub</span>`? 

cc @marcysutton is that best practice here?

wardpeet replied:
I'm pretty sure they do the same thing but it's better to not repeat yourself so 👍 for `aria-labelledby`

kbrgl replied:
Sure 👍

kbrgl commented:
@m-allanson the text doesn't fit at 320px, so I've added a media query to hide it at widths of 385px and below. Here's a look at the text at just under 385px:

![screenshot_2019-02-27 gatsby-plugin-sharp](https://user-images.githubusercontent.com/13417496/53506587-45379f00-3adc-11e9-8b8d-b7a621f62249.png)

And just over:

![screenshot_2019-02-27 gatsby-plugin-sharp 1](https://user-images.githubusercontent.com/13417496/53506628-64cec780-3adc-11e9-9f5b-ce8e861f7f21.png)

I've fixed the other issues.

fk commented:
Hey @kbrgl, thanks so much for this PR — I completely agree with the changes, 💯 to both.

Wondering if we should slightly adjust the "See starters that use this" to "See starters using this"?

In regards to the small screen issues: Looking at #11753 incoming, I think we won't get around breaking/wraping items into rows instead of hiding labels. We also want to unfiy link styles 😅, but IMO that's a thing for another PR.

kbrgl commented:
Hi, @fk!

Here's how that'll look:

![screenshot_2019-02-27 gatsby-plugin-react-helmet](https://user-images.githubusercontent.com/13417496/53524018-6318fa00-3b04-11e9-86c8-c00b22125d33.png)

fk commented:
@kbrgl #12253

marcysutton commented on www/src/components/package-readme.js:
```diff
@@ -27,22 +27,35 @@ const PackageReadMe = props => {
           href={githubUrl}
-          aria-label={`${packageName} source`}
-          title={`View source on GitHub`}
+          aria-labelledby="github-link-label"
```
For future reference, `aria-labelledby` isn't needed on a link if it contains text content.
